### PR TITLE
Added 'exist' param to the 'zadd' method as ability setup NX or XX argument

### PR DESCRIPTION
**Pull request by mastak**
Added `exist` parameter to the `zadd` method as ability setup NX or XX argument.

### CODE REVIEW

**Comment by pcinkh** [comment]
@mastak pls, merge it, to make green: https://github.com/mastak/aioredis/pull/2

**Comment by pcinkh** [comment]
@mastak if you have time try to update from master

**Comment by mastak** [comment]
@pcinkh 
have updated. thanks for the help with tests!

File: aioredis/commands/sorted_set.py [hunk 1]
```diff
@@ -18,7 +18,10 @@ class SortedSetCommandsMixin:
     ZSET_AGGREGATE_MIN = 'ZSET_AGGREGATE_MIN'
     ZSET_AGGREGATE_MAX = 'ZSET_AGGREGATE_MAX'
 
-    def zadd(self, key, score, member, *pairs):
+    SET_IF_NOT_EXIST = 'SET_IF_NOT_EXIST'  # NX
+    SET_IF_EXIST = 'SET_IF_EXIST'  # XX
```
Comment (popravich) [inline aioredis/commands/sorted_set.py]: Ouch, these two constants will conflict with the same ones in `aioredis/commands/string.py`.
Please rename both to `ZSET_IF_NOT_EXIST / ZSET_IF_EXIST`

**Comment by popravich** [comment]
Thanks,

**Pull request merged**